crates/compiler/test_gen/src/helpers/llvm.rs

```diff
@@ -335,6 +335,9 @@ pub fn helper<'a>(
     let (main_fn_name, delayed_errors, module) =
         create_llvm_module(arena, src, config, context, target, function_kind);
 
+    // for debugging:
+    //module.print_to_file(std::path::Path::new("/home/username/roc/llvm_ir.ll")).unwrap();
```

Comment:
why not add a debug flag (via the `roc_debug_flags` crate) for this instead? it would also make it more discoverable.